iset.mm

```diff
@@ -142450,6 +142508,29 @@ is a function if the structure (without the empty set) is a function.
       WNXMXKSWKWOUUJWLWNWKYTWOUUJJBDEFHOWDUQVRYFWQIWEWMWNWOWHACXFOGOUHWGWI $.
   $}
 
+  ${
+    $d A a b $.  $d B b $.  $d C a b $.  $d D a b $.  $d S a b $.  $d a b ph $.
+    setscomd.a $e |- ( ph -> A e. Y ) $.
+    setscomd.b $e |- ( ph -> B e. Z ) $.
+    setscomd.s $e |- ( ph -> S e. V ) $.
+    setscomd.ab $e |- ( ph -> A =/= B ) $.
+    setscomd.c $e |- ( ph -> C e. W ) $.
+    setscomd.d $e |- ( ph -> D e. X ) $.
+    $( Component-setting is commutative when the x-values are different.
```

Comment:
```suggestion
    $( Component-setting can be done in any order when the setted components are different.
```
I think "setted" can be tolerated as the past participle "set" when the latter is too ambiguous ?  But you'll decide.

Reply:
`Different components can be set in any order`

Reply:
I was going to suggest a different rewrite, but agree that the original sentence isn't very good wording with either "set" or "setted". I'll adopt the "Different components can be set in any order" suggestion.

Reply:
Done.